.github/workflows/run-fullnode-sync.yaml

```diff
@@ -35,7 +35,7 @@ on:
 
 jobs:
   fullnode-sync:
-    runs-on: high-perf-docker-with-local-ssd
+    runs-on: medium-perf-local-ssd
```

Comment:
this should be `medium-perf-docker-with-local-ssd` actually. It might be a bit confusing if you're looking at the deployment spec, but to clarify:
* `medium-perf-docker-with-local-ssd` is the name of the Runner.
* `medium-perf-local-ssd` is the name of the NodePool, on which the Runners are scheduled

Reply:
Aah, that makes sense! Thanks @rustielin 😄